package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "LBRY",
+  "name": "lbry-app",
```

Comment:
why did this change? will this affect the .deb package that's built? if the name changes, i think ubuntu will think its a different package...

Reply:
I changed the name to follow the [norms](https://yarnpkg.com/lang/en/docs/package-json/#toc-name).  I'm not sure how exactly it'll behave in Ubuntu since I didn't test on it, but it should normally use the [`productName` field](https://yarnpkg.com/lang/en/docs/package-json/#toc-name) declared in the `electron-builder.json` file and not the `package.json`'s `name`.

I'll test it to be sure.

Reply:
Debian does identify the app as a new package and doesn't replace the one with the previous name.

I think we should wait for the AppImage switch, which should come with the auto-update PR, before applying this change and then request Linux users that they uninstall their previous install.